### DISCUSSION TITLE
controller support

### DISCUSSION THREAD

**Discussion by veganjin** [Heroic-Games-Launcher/HeroicGamesLauncher#902] [General]
hi i love the app and excited its been working for me as i have anazon orime gaming and lots of games are free! i just tried downloading wwz aftermath and it works online and all! but its not detecting my ps3 controller…. rise of the tomb raider did (once)  anyway….. any sort of help would be greatly appreciated thank you all so much

**Comment by evil77hoelle** [comment]
Did that yesterday with my epic games version of Tomb Raider with a ps4 controller. After searching online, this is my preferred fix; (tested it also with Dead Island 2 from epic games)

(game)Settings - ADVANCED - Environment Variables -
Variable Name = PROTON_PREFER_SDL
Value = 1


I hope this helps you
